src/main/java/com/codejoust/main/service/RoomService.java

```diff
@@ -90,6 +97,14 @@ public RoomDto joinRoom(String roomId, JoinRoomRequest request) {
             user.setUserId(utility.generateUniqueId(UserService.USER_ID_LENGTH, Utility.USER_ID_KEY));
         }
 
+        // Set the account associated with the host, if any.
+        if (token.length() == 0) {
```

Comment:
if you do end up setting required = false on those request headers, I think this (and createRoom/updateUserAccount) will need to be updated to check for the null condition as well

Reply:
Done! ✅